specifyweb/frontend/js_src/lib/components/Statistics/AddStatDialog.tsx

```diff
@@ -43,6 +70,10 @@ export function AddStatDialog({
       header={statsText.chooseStatistics()}
       onClose={handleClose}
     >
+      <div>
```

Comment:
(optional) replace this with a button in the dialog (i.e, in `<Dialog buttons={...}`)